draft-ietf-tls-tls13.md

```diff
@@ -1370,6 +1370,11 @@ ClientHello (without modification) except:
 If a server receives a ClientHello at any other time, it MUST send
 a fatal "unexpected_message" alert and close the connection.
 
+If a server receives a TLSv1.3 ClientHello inside already established TLS
+session (as would be the case of the renegotiation mechanism from previous
+versions of TLS protocol), it MUST send a fatal "illegal_parameter" alert and
+close the connection.
```

Comment:
unexpected_message indicates that the message can't be sent at this point in time, since it's TLSv1.2 session, sending ClientHello messages is valid, the problem are the contents of the message.